assets/floormaps/hall/HallFloorX.js

```diff
@@ -0,0 +1,874 @@
+          strokeWidth={0}
+          stroke="#000"
+        >
+          {"H" + floorNumber + "60.03"}
+        </Text>
+        <Text
+          fontFamily="'Trebuchet MS', Gadget, sans-serif"
+          fontSize={12}
+          y={541.8}
+          x={223}
+          strokeWidth={0}
+          stroke="#000"
+        >
+          {"H" + floorNumber + "60.05"}
+        </Text>
+        <Text
+          fontFamily="'Trebuchet MS', Gadget, sans-serif"
+          fontSize={12}
+          y={541.8}
+          x={283}
+          strokeWidth={0}
+          stroke="#000"
+        >
+          {"H" + floorNumber + "60.06"}
+        </Text>
+        <Text
+          fontFamily="'Trebuchet MS', Gadget, sans-serif"
+          fontSize={12}
+          y={474.8}
+          x={284}
+          strokeWidth={0}
+          stroke="#000"
+        >
+          {"H" + floorNumber + "60.04"}
+        </Text>
+        <Path
+          d="M431.243 339.823c0-1.648 1.394-2.982 3.115-2.982 1.72 0 3.115 1.334 3.115 2.982 0 1.647-1.394 2.981-3.115 2.981-1.721 0-3.115-1.334-3.115-2.981zm10.473 8.439v-1.506c0-1.584-1.341-2.868-2.997-2.868h-8.732c-1.655 0-2.997 1.284-2.997 2.868v1.506c-.003.034-.005.07-.005.105v8.672c0 .674.57 1.22 1.275 1.22.703 0 1.275-.546 1.275-1.22v-8.544h.854v9.765h.006v13.798c0 .896.761 1.625 1.7 1.625.938 0 1.699-.728 1.699-1.625V358.26h1.12v13.798c0 .896.762 1.625 1.7 1.625s1.699-.728 1.699-1.625V358.26h.005v-9.765h.855v8.544c0 .674.572 1.22 1.275 1.22.704 0 1.274-.546 1.274-1.22v-8.673c0-.036-.003-.07-.006-.104h0z"
+          fillOpacity="null"
+          strokeOpacity="null"
+          strokeWidth={1.5}
+          stroke="#000"
+          fill="#fff"
+        />
+        <Path
+          d="M413.572 341.873c1.767 0 3.202-1.376 3.202-3.074 0-1.699-1.435-3.076-3.202-3.076-1.77 0-3.205 1.377-3.205 3.076 0 1.698 1.435 3.074 3.205 3.074zm8.981 13.257l-2.874-10.231a1.261 1.261 0 00-.11-.264c-.327-.958-1.264-1.652-2.371-1.652h-7.274c-1.154 0-2.123.753-2.409 1.776-.019.045-.035.092-.05.14l-2.822 10.232c-.188.67.227 1.36.927 1.54.698.18 1.416-.218 1.605-.89l2.216-8.036h.924l-4.023 14.541h3.79v10.04c0 .771.652 1.397 1.457 1.397.804 0 1.456-.625 1.456-1.398v-10.039h1.158v10.04c0 .771.652 1.397 1.458 1.397.803 0 1.456-.625 1.456-1.398v-10.039h3.789l-4.038-14.541h.948l2.257 8.037c.188.67.906 1.069 1.605.89.697-.181 1.112-.871.925-1.542h0z"
+          fillOpacity="null"
+          strokeOpacity="null"
+          strokeWidth={1.5}
+          stroke="#000"
+          fill="#fff"
+        />
+        <Use
+          xlinkHref="#prefix__a"
+          transform="matrix(.04757 0 0 .06771 353.073 653.876)"
+          y={-2360.53}
+          x={2298.318}
+        />
+        <Path
+          strokeLinecap="null"
+          strokeLinejoin="null"
+          strokeOpacity="null"
+          strokeWidth={1.5}
+          stroke="#000"
+          fill="none"
+          d="M459.538 426.23h53.846M514.153 611.23h-54.615"
+        />
+      </G>
+      <HallClass floor={8} />
```

Comment:
Just noticed this. Will change it after your comments. Should be `<HallClass floor={floorNumber} />` instead.